src/index.js

```diff
@@ -197,7 +195,8 @@ class MiniCssExtractPlugin {
                   renderedModules,
                   compilation.runtimeTemplate.requestShortener
                 ),
-              filenameTemplate: this.options.filename,
+              filenameTemplate:
+                this.options.filenameTemplate || this.options.filename,
```

Comment:
No need new function here you should use `filename: () => { /* logic for nameing */ }`

Reply:
good idea. Should we also only pass `{chunk}`?

Reply:
`filenameTemplate` now called from an anonymous function